### PR TITLE
change the organization to io.higherkindness

### PR DESCRIPTION
**Pull request by pepegar**
otherwise it will try to pubish the artifact as "higherkindness" %% "skeuomorph"

### CODE REVIEW

**Comment by pepegar** [comment]
Relevant conversation: https://github.com/higherkindness/skeuomorph/pull/20#discussion_r230595219

Locally at least, it publishes to "~/.ivy2/local/higherkindnes" instead of  "~/.ivy2/local/io.higherkindnes"

File: build.sbt [hunk 1]
```diff
@@ -71,7 +71,7 @@ pgpSecretRing := file(s"$gpgFolder/secring.gpg")
 
 // General Settings
 lazy val commonSettings = Seq(
-  organization := "higherkindness",
+  organization := "io.higherkindness",
```
Comment (juanpedromoreno) [inline build.sbt]: This setting can be removed, you need to add this one: https://github.com/higherkindness/catamorph/pull/3/files#diff-fdc3abdfd754eeb24090dbd90aeec2ceR61 ;)

Comment (juanpedromoreno) [reply build.sbt]: The groupId is being set up in this way: https://github.com/higherkindness/catamorph/pull/3/files#diff-fdc3abdfd754eeb24090dbd90aeec2ceR65

Comment (pepegar) [reply build.sbt]: Thanks!!

**Comment by pepegar** [comment]
ready for re-review :)

**Pull request merged**